app/frontend/Calendar/DisplayEvent/Invitation.svelte

```diff
@@ -14,88 +14,67 @@
       {ex?.message ?? ex}
     {/await}
   {/if}
-  {#if message.scheduling == Scheduling.Request}
-    <hbox class="buttons">
+  <hbox class="buttons">
+    {#if selectedCalendar}
+      <AccountDropDown selectedAccount={selectedCalendar} accounts={calendars} withIcon={true} withLabel={true} disabled={calendars.length < 2} />
+    {/if}
+    {#if message.scheduling == Scheduling.Request}
       <Button
         label={$t`Confirm *=> Confirm to attend the meeting`}
         onClick={onAccept}
         icon={AcceptIcon}
         classes="accept" />
+      <Button
+        label={$t`Maybe *=> Maybe attend the meeting`}
+        onClick={onTentative}
+        icon={MaybeIcon}
+        classes="maybe secondary" />
       <Button
         label={$t`Reject *=> Decline to attend the meeting`}
         onClick={onDecline}
         icon={DeclineIcon}
         classes="decline secondary" />
-      <ButtonMenu buttonIcon={ChevronDownIcon}>
-        <MenuItem
-          label={$t`Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe" />
-      </ButtonMenu>
-    </hbox>
-  {:else if message.scheduling == Scheduling.Accepted}
-    <hbox class="buttons">
-      <Button
-        label={$t`Confirmed *=> A meeting request has been confirmed by you`}
-        icon={AcceptIcon}
-        selected={true}
-        disabled={true}
-        classes="accept done" />
-      <ButtonMenu buttonIcon={ChevronDownIcon}>
-        <MenuItem
-          label={$t`Change to Reject *=> not attend the meeting`}
-          onClick={onDecline}
-          icon={DeclineIcon}
-          classes="decline" />
-        <MenuItem
-          label={$t`Change to Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe" />
-      </ButtonMenu>
-    </hbox>
-  {:else if message.scheduling == Scheduling.Declined}
-    <hbox class="buttons">
-      <Button
-        label={$t`Rejected *=> A meeting request has been declined by you`}
-        icon={DeclineIcon}
-        selected={true}
-        disabled={true}
-        classes="decline done" />
-      <ButtonMenu buttonIcon={ChevronDownIcon}>
-        <MenuItem
-          label={$t`Change to Accept *=> attend the meeting`}
-          onClick={onAccept}
-          icon={AcceptIcon}
-          classes="accept" />
-        <MenuItem
-          label={$t`Change to Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe" />
-      </ButtonMenu>
-    </hbox>
-  {/if}
```

Comment:
We want to show how our user answered to a given invitation email. Please keep the UI.

Reply:
Sadly it doesn't actually work, at least, not in the state it's in right now. Leaving it here would also break the fix I'm trying to make. And fixing it is beyond the scope of this PR.

Reply:
> Leaving it here would also break the fix I'm trying to make.

How it would break with this PR, and why?

We've worked on this UI quite a lot. In fact, 3 people in the team worked together on this. The translated labels were pain to get right, and needed several rounds to be correctly translated in various languages. I do no want that work to get lost. Please consulting with me before making such drastic changes to the UI.

At the very least, please wrap them in an `{#if false}`, and then follow up with another PR to fix it and make them functional.

Reply:
FWIW, the UI specs are here: https://parula.cloud.xwiki.com/xwiki/bin/view/Main/Calendar/Email%20invitation/

Reply:
Because Scheduling.Accepted and Scheduling.Declined have always referred to incoming messages from attendees that you invited and have nothing whatsoever to do with how you responded to an invitation.

Reply:
Ah, OK, then I misunderstood the API. We need to straighten that out, to make it clearer.
`{#if false}` is fail, then. Thanks that you did that and restored the buttons.